D:
- 0.03697415024499557
- -0.0440168161187022
- 0.00036042073175322634
- 0.0037431576602881076
- 0.0
K:
- 299.75546813696866
- 0.0
- 297.9828957820265
- 0.0
- 299.79306962788024
- 236.7508004173323
- 0.0
- 0.0
- 1.0
P:
- 293.7798156738281
- 0.0
- 297.9842257751334
- 0.0
- 0.0
- 301.0251159667969
- 236.38756442853628
- 0.0
- 0.0
- 0.0
- 1.0
- 0.0
R:
- 1.0
- 0.0
- 0.0
- 0.0
- 1.0
- 0.0
- 0.0
- 0.0
- 1.0
distortion_model: plumb_bob
height: 480
width: 640
